### PR TITLE
Improving login page

### PR DESCRIPTION
**Pull request by olivierteboul**
- mainly style
- simplifying backend (IMO)

### CODE REVIEW

File: icubam/backoffice/handlers/login.py [hunk 1]
```diff
@@ -7,32 +7,23 @@ class LoginHandler(BaseHandler):
...
+    email = self.get_body_argument("email", "")
+    password = self.get_body_argument("password", "")
+    userid = self.store.auth_user(email, password)
+    if userid is not None:
       self.set_secure_cookie(self.COOKIE, tornado.escape.json_encode(userid))
       return self.redirect(self.get_argument("next", "/"))
-    # Failed to login, GET page again with the error message.
-    return self.redirect(self.ROUTE + self._makeErrorMessage(error))
+
+    return self.redirect(f"{self.ROUTE}?error=true")
```
Comment (fpquintao) [inline icubam/backoffice/handlers/login.py:29]: I didn't know this syntax. Is it the new format?

File: icubam/backoffice/handlers/login.py [hunk 2]
```diff
@@ -7,32 +7,23 @@ class LoginHandler(BaseHandler):
 
   ROUTE = "/login"
 
-  def _makeErrorMessage(self, error):
-    return "?error={}".format(error)
+  ERROR = 'Authentication failed'
```
Comment (fpquintao) [inline icubam/backoffice/handlers/login.py:10]: not used?

File: icubam/backoffice/templates/login.html [hunk 3]
```diff
@@ -1,42 +1,76 @@
...
+<div class="login-box">
+  <div class="login-logo">
+    <b>ICUBAM</b> admin</a>
+  </div>
+  <!-- /.login-logo -->
+  <div class="card">
+    <div class="card-body login-card-body">
+      <p class="login-box-msg">{{_("Sign in to start your session")}}.</p>
+      {% if error %}
+      <p class="login-box-msg text-danger">{{_("Authentication failed")}}.</p>
```
Comment (fpquintao) [inline icubam/backoffice/templates/login.html:31]: nice, this is better for i18n

File: icubam/backoffice/templates/login.html [hunk 4]
```diff
@@ -1,42 +1,76 @@
...
+          <!-- <input type="email" class="form-control" placeholder="Email" name="email"> -->
+          <input class="form-control" placeholder="Email" name="email">
+          <div class="input-group-append">
+            <div class="input-group-text">
+              <span class="fas fa-envelope"></span>
+            </div>
+          </div>
+        </div>
+        <div class="input-group mb-3">
+          <input type="password" class="form-control" placeholder="Password" name="password">
```
Comment (fpquintao) [inline icubam/backoffice/templates/login.html:45]: add required

Comment (fpquintao) [inline icubam/backoffice/templates/login.html:45]: I never now how post works exactly, should we have "name" and "id" in the tag, or just name is enough?

File: icubam/backoffice/templates/login.html [hunk 5]
```diff
@@ -1,42 +1,76 @@
...
+      {% if error %}
+      <p class="login-box-msg text-danger">{{_("Authentication failed")}}.</p>
+      {% end %}
 
-  <!-- Custom styles for this template -->
-  <link href="signin.css" rel="stylesheet">
+      <form action="/login" method="post" class="form-signin">
+        <div class="input-group mb-3">
+          <!-- <input type="email" class="form-control" placeholder="Email" name="email"> -->
+          <input class="form-control" placeholder="Email" name="email">
```
Comment (fpquintao) [inline icubam/backoffice/templates/login.html:37]: add required for some quick client side validation. Also autofocus (helps a lot in mobile)

File: icubam/backoffice/handlers/login.py (hunk 1 above)
Comment (olivierteboul) [reply icubam/backoffice/handlers/login.py:29]: yes ! format is still useful when your have complex things to put though

File: icubam/backoffice/templates/login.html (hunk 4 above)
Comment (olivierteboul) [reply icubam/backoffice/templates/login.html:45]: i guess the name is enough.but not totally sure (it works like this)

**Pull request merged**